acceptance/configurations_test.go

```diff
@@ -403,13 +403,13 @@ var _ = Describe("Configurations", LConfiguration, func() {
 		It("doesn't unbind a service-owned configuration", func() {
 			out, err := env.Epinio("", "configuration", "unbind", config, appName)
 			Expect(err).To(HaveOccurred(), out)
-			Expect(out).To(ContainSubstring("Bad Request: Configuration belongs to service"))
+			Expect(out).To(ContainSubstring("Bad Request: Configuration '" + config + "' belongs to service"))
```

Comment:
```suggestion
			Expect(out).To(ContainSubstring("Bad Request: Configuration '%s' belongs to service", config))
```

Reply:
Does `ContainSubstring` handle printf-style placeholders ?
I suspect a `fmt.Sprintf` should be there. Will make the change.

As for tests of `--all`. I left them out intentionally.

Having them are, I strongly believe, a bad idea.

CI has many tests running concurrently on the same E install. A test invoking any kind of `delete --all` will very likely interfere with all tests running in parallel to it, i.e. their services, applications, and configurations, i.e. break them. And note that for a suitable user, i.e. admin, `namespace delete --all` is just one step short of E uninstall.